src/domain/payments/index.types.d.ts

```diff
@@ -11,7 +11,18 @@ type XorPaymentHashProperty = XOR<
   { intraLedgerHash: IntraLedgerHash }
 >
 
-type PaymentFlowCommonState<S extends WalletCurrency, R extends WalletCurrency> = {
+type AmountsAndFees = {
+  btcPaymentAmount: BtcPaymentAmount
+  btcProtocolFee: BtcPaymentAmount
+
+  usdPaymentAmount: UsdPaymentAmount
+  usdProtocolFee: UsdPaymentAmount
+}
```

Comment:
please add a comment to clarify that these are protocol + bank fees (while we decide this at ACE meeting)

Reply:
Thinking about this again, we can probably just rename these in the code. I've made a card for this in the project to do next
https://github.com/GaloyMoney/galoy/projects/9#card-87595055